### PR TITLE
Mongo

### PR DESCRIPTION
**Pull request by brianorwhatever**

### CODE REVIEW

File: backend/src/admin/admin.controller.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Controller, Get, Post } from '@nestjs/common';
+import { DBService } from 'src/db/db.service';
+
+@Controller('/admin')
+export class AdminController {
+  constructor(private db: DBService) {}
+
+  @Post('/drop-db')
```
Comment (Vanderscycle) [inline backend/src/admin/admin.controller.ts:8]: I think we should have a production flag, this is a dangerous command that is easily accessible

File: backend/src/db/db.service.ts [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+      .db(this.dbName)
+      .collection(this.collection)
+      .findOne({ _id: docId });
+    delete finalRes._id;
+    return finalRes;
+  }
+
+  // insert, even if duplicate
+  async insert(obj) {
+    if (obj['@id'].length !== 32 || obj['@id'].substr(0, 3) !== 'a51') {
```
Comment (Vanderscycle) [inline backend/src/db/db.service.ts:209]: should we use class validator for that? https://github.com/typestack/class-validator

**Review by Vanderscycle — APPROVED** [review]

File: backend/src/admin/admin.controller.ts (hunk 1 above)
Comment (brianorwhatever) [reply backend/src/admin/admin.controller.ts:8]: yep definitely will need that or remove this once not in such a fluid development moed

File: backend/src/db/db.service.ts (hunk 2 above)
Comment (brianorwhatever) [reply backend/src/db/db.service.ts:209]: yeah that's definitely the direction we should head. this is actually old from the old project

**Pull request merged**